packages/api/test/integration.test.ts

```diff
@@ -143,14 +140,12 @@ describe('integration tests', function () {
         expect(res.requestBody.split(`--${res.boundary}`).filter(Boolean)).to.deep.equal([
           '\r\nContent-Disposition: form-data; name="orderId"\r\n\r\n1234\r\n',
           '\r\nContent-Disposition: form-data; name="userId"\r\n\r\n5678\r\n',
-          isNode18
```

Comment:
I no longer need all this mess because the changes that broken this stuff in [undici](https://npm.im/undici) were reverted and then published as part of Node 18.4.

Reply:
I'm glad undici cleaned their mess up.